src/app-country-search.js

```diff
@@ -0,0 +1,96 @@
+import axios from "axios";
+
+const countryContainer = document.getElementById("country-information");
+let userInput = "";
+
+//After search button is pressed, the user input will be captured
+document.getElementById("search-button").addEventListener("click", getUserInput);
+
+//Capture user input
+function getUserInput(ev) {
+    ev.preventDefault();
+    userInput = document.getElementById("input-field").value;
+
+    getCountryByName();
+}
+
+//get Country by name and use userInput information for API URL
+async function getCountryByName() {
+    document.getElementById("country-information").innerHTML = "";                                             //Clear previous elements
+
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${userInput}`);
+
+        console.log(result.data[0]);
+        displayFlag(result);
+        displayCountryName(result);
+        displayCountryInfo(result);
+        displayCapitalInfo(result);
+        displayLanguages(result);
```

Comment:
In elke functie doe je `result.data[0]`, kun je dat niet beter hier doen en de waarde dan meegeven aan de functies